libs/sdk-ui-dashboard/src/model/store/config/configSelectors.ts

```diff
@@ -128,6 +128,15 @@ export const selectIsExport = createSelector(selectConfig, (state) => {
     return state.isExport;
 });
 
+/**
+ * Returns whether the Dashboard is white labeled.
+ *
+ * @internal
+ */
+export const selectIsWhiteLabeled = createSelector(selectConfig, (state) => {
+    return state.isWhiteLabeled;
```

Comment:
Should we add a default value here? Something like `?? false`?

Reply:
It is not necessary, default value is set in [initializeDashboardHandler/resolveDashboardConfig.ts](https://github.com/gooddata/gooddata-ui-sdk/pull/2613#diff-d11e08af68868d9ccf444ce9827c7a8a0270faa1dbd8fbf91d8697f1361103ef)